zebra-network/src/peer_set/candidate_set/tests/prop.rs

```diff
@@ -21,3 +31,62 @@ proptest! {
         }
     }
 }
+
+proptest! {
+    #![proptest_config(ProptestConfig::with_cases(16))]
```

Comment:
How long does this test take?

We try to keep individual test runtime under 20 seconds on most machines. So if these tests take longer than 10 seconds, let's halve the number of cases.

Reply:
Worst case is (4 * 100 + 400 + 4 * 100) ms = 1.2 s per test. Best case is 400 ms. For 16 tests that means the minimum is 6.4 s and the maximum is 19.2 s. It's usually less than that though, and after a few runs on my machine it seems to take around 9 seconds on average. Should I still reduce the number of cases?

Reply:
10-20 seconds is fine, there are quite a few tests that take 10-20 seconds even on fast machines.